sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentViewModel.kt

```diff
@@ -115,6 +115,21 @@ internal class AuthorizingPaymentViewModel(
         }
     }
 
+    private suspend fun getConfigs(): NetceteraConfig? {
```

Comment:
Let's move `getConfigs` and createNetceteraConfigUrl` to a separate config class. ViewModel just needs to hold the data for the UI. It doesn't need to know where the data comes from.

Reply:
but we also have the `sendAuthenticationRequest` function that gets data for authentication also. That is why i kept the other functions.

Reply:
I understand it but if we continue doing this then we will keep on adding more code that should be here. There might be more stuff here that shouldn't be here but we can start by focusing on Netcetra config.